SimCalorimetry/EcalEBTrigPrimProducers/python/ecalEBTriggerPrimitivePhase2Digis_cfi.py

```diff
@@ -0,0 +1,16 @@
+import FWCore.ParameterSet.Config as cms
+
+#
+# attention: default is changed to work on unsuppressed digis!! ##############
+#
+
+simEcalEBTriggerPrimitivePhase2Digis = cms.EDProducer("EcalEBTrigPrimPhase2Producer",
+    BarrelOnly = cms.bool(True),
+    barrelEcalDigis = cms.InputTag("simEcalUnsuppressedDigis"),
+    binOfMaximum = cms.int32(8), ## optional from release 200 on, from 1-10
+    Famos = cms.bool(False),
+    TcpOutput = cms.bool(False),
+    Debug = cms.bool(False)
+)
+
+
```

Comment:
It there would be a `fillDescriptions` this could be changed to
```suggestion
import  SimCalorimetry.EcalEBTrigPrimProducers.ecalEBTriggerPrimitivePhase2Digis_cfi as _mod

simEcalEBTriggerPrimitivePhase2Digis = _mod.ecalEBTriggerPrimitivePhase2Digis.clone()
#
# add modifications from fillDescriptions defaults here
#
```

Reply:
I do not agree. I do not to clone anything. There is just one module. IF in the future there will be the need of multiple producers, then we will do that. I have done this in the past in different contexts